home-server-generator/src/main/resources/mustaches/server/apiServiceImpl.mustache

```diff
@@ -21,9 +20,8 @@ import javax.validation.constraints.*;
 public class {{classname}}ServiceImpl implements {{classname}}Service {
     {{#operation}}
     @Override
-    public Response {{nickname}}({{#allParams}}{{>serviceQueryParams}}{{>servicePathParams}}{{>serviceHeaderParams}}{{>serviceBodyParams}}{{>serviceFormParams}}, {{/allParams}}SecurityContext securityContext) throws NotFoundException {
-        // do some magic!
-        return Response.ok().entity(new ApiResponseMessage(ApiResponseMessage.OK, "magic!")).build();
+    public Response {{nickname}}({{#allParams}}{{>serviceQueryParams}}{{>servicePathParams}}{{>serviceHeaderParams}}{{>serviceBodyParams}}{{>serviceFormParams}}, {{/allParams}}SecurityContext securityContext) {
```

Comment:
Please try wrap comma with `{{#comma}},{{/comma}}`

Reply:
https://github.com/OpenAPITools/openapi-generator/blob/master/docs/templating.md#firstlast

Reply:
Cool! I will try to use this solution, thanks